### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ValentinoTheGamer**
instead of mdt imade it /pl, because of my country. and added tablet animation

### CODE REVIEW

File: disc-mdt/client/main.lua [hunk 1]
```diff
@@ -22,7 +22,8 @@ end)
 
 local isShowing = false
 
-RegisterCommand('mdt', function()
+RegisterCommand('pl', function()
```
Comment (DiscworldZA) [inline disc-mdt/client/main.lua]:
```suggestion
RegisterCommand('mdt', function()
```

File: disc-mdt/client/main.lua [hunk 2]
```diff
@@ -47,8 +49,37 @@ RegisterCommand('mdt', function()
     end
 end)
 
+local tijdelijk = '0'
```
Comment (DiscworldZA) [inline disc-mdt/client/main.lua]:
```suggestion
local temp = false
```

File: disc-mdt/client/main.lua [hunk 3]
```diff
@@ -47,8 +49,37 @@ RegisterCommand('mdt', function()
...
+function startAnim()
+	Citizen.CreateThread(function()
+    
+      if tijdelijk ~= '1' then
```
Comment (DiscworldZA) [inline disc-mdt/client/main.lua]:
```suggestion
      if not temp then
```

File: disc-mdt/client/main.lua [hunk 4]
```diff
@@ -47,8 +49,37 @@ RegisterCommand('mdt', function()
...
+	       RequestAnimDict("amb@world_human_seat_wall_tablet@female@base")
+              while not HasAnimDictLoaded("amb@world_human_seat_wall_tablet@female@base") do
+                Citizen.Wait(0)
+              end
+		  attachObject()
+		  TaskPlayAnim(GetPlayerPed(-1), "amb@world_human_seat_wall_tablet@female@base", "base" ,8.0, -8.0, -1, 50, 0, false, false, false)
+          tijdelijk = '1'
```
Comment (DiscworldZA) [inline disc-mdt/client/main.lua]:
```suggestion
          temp = true
```

File: disc-mdt/client/main.lua [hunk 5]
```diff
@@ -47,8 +49,37 @@ RegisterCommand('mdt', function()
...
+	end)
+end
+
+function attachObject()
+	tab = CreateObject(GetHashKey("prop_cs_tablet"), 0, 0, 0, true, true, true)
+	AttachEntityToEntity(tab, GetPlayerPed(-1), GetPedBoneIndex(GetPlayerPed(-1), 57005), 0.17, 0.10, -0.13, 20.0, 180.0, 180.0, true, true, false, true, 1, true)
+end
+
+function stopAnim()
+    tijdelijk = '0'
```
Comment (DiscworldZA) [inline disc-mdt/client/main.lua]:
```suggestion
    temp = false
```

**Comment by ValentinoTheGamer** [comment]
Good thanks voor looking

**Pull request merged**